README.md

```diff
@@ -1,6 +1,6 @@
 # Medic Mobile
 
-These instructions should help you get setup to run or develop on Medic Mobile's Community Health Application Framework. For latest changes and release announcements see our [release notes](https://github.com/medic/medic/tree/master/release-notes).
+These instructions should help you get setup to run or develop on Medic Mobile's Core Framework. For latest changes and release announcements see our [release notes](https://github.com/medic/medic/tree/master/release-notes).
```

Comment:
I think the "Core Framework" is in relation to the CHT, so perhaps something like this:
```suggestion
These instructions should help you get setup to run or develop on the Community Health Toolkit's Core Framework. For latest changes and release announcements see our [release notes](https://github.com/medic/medic/tree/master/release-notes).
```

Making this change may require other changes in the README though.